modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -2,62 +2,110 @@
 ====
 
- See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for additional examples of the NMState syntax.
+.Prerequisites
 
+. For each node being configured with a static IP, you must configure a DNS PTR record containing a valid host name including the specific IP associated with that node.
+
+.Procedure
+
+. (Optional) Consider saving the `networkConfig` YAML syntax to a file.
+
+.. Test it using the NMState command line interface. Do this before including it in the `install-config.yaml` file, because the installation program will not check the NMState YAML syntax. 
+
+.. Execute `nmstatectl gc <yaml-config>` to test the syntax. Errors in the YAML syntax might result in a failure to apply the network configuration. 
+
+.. Maintain the valid YAML syntax when applying changes using Kubernetes NMState after deployment or when expanding the cluster.
+
+. To use the `networkConfig` configuration setting, you must provide an NMState YAML configuration. In the following example, the user name is `user` and the password is `password`:
++
 .Example
-[source,yaml]
+[source,terminal]
 ----
   hosts:
         - name: openshift-master-0
           role: master
           bmc:
-            address: redfish+http://<out-of-band-ip>/redfish/v1/Systems/
-            username: <user>
-            password: <password>
+            address: redfish+http://_<out-of-band-ip>_/redfish/v1/Systems/
+            username: _<user>_
+            password: _<password>_
             disableCertificateVerification: null
-          bootMACAddress: <NIC1_mac_address>
+          bootMACAddress: _<NIC1_mac_address>_
           bootMode: UEFI
           rootDeviceHints:
             deviceName: "/dev/sda"
           networkConfig: <1>
             interfaces:
-            - name: <NIC1_name>
+            - name: _<NIC1_name>_
               type: ethernet
               state: up
               ipv4:
                 address:
-                - ip: "<IP_address>"
+                - ip: "_<IP_address>_"
                   prefix-length: 24
                 enabled: true
             dns-resolver:
               config:
                 server:
-                - <DNS_IP_address>
+                - _<DNS_IP_address>_
             routes:
               config:
               - destination: 0.0.0.0/0
-                next-hop-address: <IP_address>
-                next-hop-interface: <NIC1_name>
+                next-hop-address: _<IP_address>_
+                next-hop-interface: _<NIC1_name>_
 ----
 <1> Add NMState YAML syntax to configure host interfaces.
++
+When configuring static IP addresses, be aware that while the cluster's nodes can be assigned a static IP address as part of the `install-config` file, the bootstrap VM cannot. In the case of configuring the bootstrap VM without a DHCP server, you will need to configure it using ignition, per the following example:
```

Comment:
**per** _preposition_
Do not use to mean "according to". Use a phrase such as "according to", "as", or "as in".

```suggestion
When configuring static IP addresses, be aware that while the cluster's nodes can be assigned a static IP address as part of the `install-config` file, the bootstrap VM cannot be assigned a static IP address. In the case of configuring the bootstrap VM without a DHCP server, you must configure it using Ignition, as shown in the following example:
```